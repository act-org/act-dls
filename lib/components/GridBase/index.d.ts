/**
 * @prettier
 */
import * as React from 'react';
import { GridProps } from '@material-ui/core';
export declare type Props = GridProps;
declare const GridBase: React.FC<Props>;
export default GridBase;
//# sourceMappingURL=index.d.ts.map