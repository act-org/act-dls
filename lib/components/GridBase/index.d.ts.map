{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../../../src/components/GridBase/index.tsx"],"names":[],"mappings":"AAAA;;GAEG;AAEH,OAAO,KAAK,KAAK,MAAM,OAAO,CAAC;AAC/B,OAAO,EAAQ,SAAS,EAAE,MAAM,mBAAmB,CAAC;AAEpD,oBAAY,KAAK,GAAG,SAAS,CAAC;AAE9B,QAAA,MAAM,QAAQ,EAAE,KAAK,CAAC,EAAE,CAAC,KAAK,CAE7B,CAAC;AAEF,eAAe,QAAQ,CAAC"}