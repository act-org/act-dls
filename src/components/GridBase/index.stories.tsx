/**
 * @prettier
 */

import * as React from 'react';

import GridBase from '.';

export const base = (): React.ReactElement<any> => (
  <GridBase>
    <div>Grid Base</div>
  </GridBase>
);

export default {
  component: GridBase,
  parameters: {
    info: {
      text: 'This is the base variant of the Grid component.',
    },
  },
  title: 'Atoms/Grid',
};
