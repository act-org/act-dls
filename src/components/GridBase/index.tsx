/**
 * @prettier
 */

import * as React from 'react';
import { Grid, GridProps } from '@material-ui/core';

export type Props = GridProps;

const GridBase: React.FC<Props> = (props: Props): React.ReactElement<any> => (
  <Grid {...props} />
);

export default GridBase;
