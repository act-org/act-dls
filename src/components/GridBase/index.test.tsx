/**
 * @prettier
 */

import * as React from 'react';

import render from '~/helpers/test/render';

import GridBase from '.';

describe('GridBase', () => {
  it('matches the snapshot', () => {
    const { container } = render(<GridBase>GridBase</GridBase>);

    expect(container).toMatchSnapshot();
  });
});
