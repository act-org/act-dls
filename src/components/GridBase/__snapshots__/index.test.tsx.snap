// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridBase matches the snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root"
  >
    GridBase
  </div>
</div>
`;
